d3a_interface/validators/load_validator.py

```diff
@@ -15,104 +15,113 @@
-                                              f"{LoadSettings.AVG_POWER_LIMIT.max}."]}
-        validate_range_limit(LoadSettings.AVG_POWER_LIMIT.min,
-                             kwargs["avg_power_W"],
-                             LoadSettings.AVG_POWER_LIMIT.max, error_message)
+    @classmethod
+    def validate_energy(cls, **kwargs):
+        """Validate energy values of the device."""
+        if "avg_power_W" in kwargs and kwargs["avg_power_W"] is not None:
+            error_message = {"misconfiguration": [f"avg_power_W should be in between "
+                                                  f"{LoadSettings.AVG_POWER_LIMIT.min} & "
+                                                  f"{LoadSettings.AVG_POWER_LIMIT.max}."]}
+            validate_range_limit(LoadSettings.AVG_POWER_LIMIT.min,
+                                 kwargs["avg_power_W"],
+                                 LoadSettings.AVG_POWER_LIMIT.max, error_message)
 
-    if (("avg_power_W" in kwargs and kwargs["avg_power_W"] is not None) or
-        ("hrs_per_day" in kwargs and kwargs["hrs_per_day"] is not None) or
-        ("hrs_of_day" in kwargs and kwargs["hrs_of_day"] is not None)) \
-            and ("daily_load_profile" in kwargs and kwargs["daily_load_profile"] is not None):
-        raise D3ADeviceException(
-            {"misconfiguration": [f"daily_load_profile shouldn't be set with "
-                                  f"avg_power_W, hrs_per_day & hrs_of_day."]})
-    if "hrs_per_day" in kwargs and kwargs["hrs_per_day"] is not None:
-        error_message = {"misconfiguration": [f"hrs_per_day should be in between "
-                                              f"{LoadSettings.HOURS_LIMIT.min} & "
-                                              f"{LoadSettings.HOURS_LIMIT.max}."]}
-        validate_range_limit(LoadSettings.HOURS_LIMIT.min,
-                             kwargs["hrs_per_day"],
-                             LoadSettings.HOURS_LIMIT.max, error_message)
-    if ("hrs_of_day" in kwargs and kwargs["hrs_of_day"] is not None) and \
-            any([not LoadSettings.HOURS_LIMIT.min <= h <= LoadSettings.HOURS_LIMIT.max
-                 for h in kwargs["hrs_of_day"]]):
-        raise D3ADeviceException(
-            {"misconfiguration": [f"hrs_of_day should be less between "
-                                  f"{LoadSettings.HOURS_LIMIT.min} & "
-                                  f"{LoadSettings.HOURS_LIMIT.max}."]})
-    if ("hrs_of_day" in kwargs and kwargs["hrs_of_day"] is not None) and \
-            ("hrs_per_day" in kwargs and kwargs["hrs_per_day"] is not None) and \
-            (len(kwargs["hrs_of_day"]) < kwargs["hrs_per_day"]):
-        raise D3ADeviceException(
-            {"misconfiguration": [f"length of hrs_of_day list should be "
-                                  f"greater than or equal hrs_per_day."]})
+        if (("avg_power_W" in kwargs and kwargs["avg_power_W"] is not None) or
```

Comment:
Same applies to the rest of similar checks
```suggestion
        if ((kwargs.get("avg_power_W") is not None) or
```